template.go

```diff
@@ -253,7 +289,7 @@ func (loader *TemplateLoader) loadIntoEngine(engine TemplateEngine, baseTemplate
 		return
 	}
```

Comment:
Add in check to see if template map contains the template already, if so do not add it
```
 	if loadedTemplate, found := runtime.TemplateMap[baseTemplate.TemplateName]; found!=false {
 		// Duplicate template found in map
 		TRACE.Println("template already exists in map: ", baseTemplate.TemplateName, " in engine ", engine.Name(), "\r\n\told file:",
                loadedTemplate.Location(), "\r\n\tnew file:", baseTemplate.FilePath)
 		return
 	}
```